Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -166,7 +166,9 @@ file that was distributed with this source code.
         {% endif %}
         <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
             {{- widget|raw -}}
-            {{- label is not same as(false) ? label|trans({}, translation_domain) -}}
+            {%- if sonata_admin.options['use_icheck'] -%}&nbsp;{%- endif -%}
```

Comment:
Originaly I wrote this fix in `Resources/public/css/styles.css` (Inspired by bootstrap)
```css
.checkbox div[class^="icheckbox"],
.checkbox-inline div[class^="icheckbox"],
.radio div[class^="icheckbox"],
.radio-inline div[class^="icheckbox"] {
    position: relative;
    margin-top: 4px \9;
    margin-left: -20px;
    margin-right: 5px;
    margin-top: -3px;
}
.form-inline .checkbox div[class^="icheckbox"],
.form-inline .checkbox-inline div[class^="icheckbox"],
.form-inline .radio div[class^="icheckbox"],
.form-inline .radio-inline div[class^="icheckbox"] {
    position: relative;
    margin-top: 4px \9;
    margin-left: 0;
    margin-right: 5px;
    margin-top: -3px;
}
```

So I can remove the `use_icheck` options test in https://github.com/sonata-project/SonataAdminBundle/blob/master/Resources/views/Form/form_admin_fields.html.twig#L83 so the widget is rendered the same whether `use_icheck` option is `true`or `false`.

What do you think @ju1ius ?